test/batch/configuration.ts.snap

```diff
@@ -0,0 +1,175 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Configuration Configuration is invalid if check_type is missing 1`] = `
+Array [
+  "ERROR:  Configuration file is invalid",
+  "[Error [ValidationError]: \\"rules[0].check_type\\" is required] {
+  _original: { rules: [ [Object] ] },
+  details: [
+    {
+      message: '\\"rules[0].check_type\\" is required',
+      path: [Array],
+      type: 'any.required',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if condition is missing 1`] = `
+Array [
+  "ERROR:  Configuration file is invalid",
+  "[Error [ValidationError]: \\"rules[0].condition\\" is required] {
+  _original: { rules: [ [Object] ] },
+  details: [
+    {
+      message: '\\"rules[0].condition\\" is required',
+      path: [Array],
+      type: 'any.required',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if locksReviewTeam is empty or missing 1`] = `
+Array [
+  "ERROR:  Locks Review Team should be provided",
+]
+`;
+
+exports[`Configuration Configuration is invalid if min_approvals is less than 1 1`] = `
+Array [
+  "ERROR:  Configuration file is invalid",
+  "[Error [ValidationError]: \\"rules[0].min_approvals\\" must be greater than or equal to 1] {
+  _original: { rules: [ [Object] ] },
+  details: [
+    {
+      message: '\\"rules[0].min_approvals\\" must be greater than or equal to 1',
+      path: [Array],
+      type: 'number.min',
+      context: [Object]
+    }
+  ]
+}",
+]
+`;
+
+exports[`Configuration Configuration is invalid if min_approvals is less than 1 2`] = `
```

Comment:
> Otherwise, two tests are similar, I do not get it.

You're right, there's a duplicated test. I put this feedback in https://github.com/paritytech/pr-custom-review/issues/44.